.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I'm assuming this file wasn't meant to be in here as you added `.vscode` to the `.gitignore`?

Reply:
Correct.. I should not have pushed that.